pints/_mcmc/_adaptive_covariance.py

```diff
@@ -154,44 +149,29 @@ def tell(self, fx):
             # Clear proposal
             self._proposed = None
 
+            # Set r to zero
+            self._log_acceptance_ratio = float('-Inf')
+
             # Return first point for chain
             return self._current
 
         # Check if the proposed point can be accepted
-        accepted = 0
-        if np.isfinite(fx):
-            u = np.log(np.random.uniform(0, 1))
-            if u < fx - self._current_log_pdf:
-                accepted = 1
-                self._current = self._proposed
-                self._current_log_pdf = fx
-
-        # Clear proposal
-        self._proposed = None
-
-        # Adapt covariance matrix
-        if self._adaptive:
-            # Set gamma based on number of adaptive iterations
-            gamma = self._adaptations ** -0.6
-            self._adaptations += 1
-
-            # Update mu, log acceptance rate, and covariance matrix
-            self._mu = (1 - gamma) * self._mu + gamma * self._current
-            self._loga += gamma * (accepted - self._target_acceptance)
-            dsigm = np.reshape(
-                self._current - self._mu, (self._n_parameters, 1))
-            self._sigma = (
-                (1 - gamma) * self._sigma + gamma * np.dot(dsigm, dsigm.T))
+        self._log_acceptance_ratio = fx - self._current_log_pdf
 
-        # Update acceptance rate (only used for output!)
```

Comment:
Again, can we write this so that the base class by itself will generate an error if used? So have some kind of _tell method that the base class calls and the subclasses implement?

Reply:
This section of the code has been removed now.